src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Constants/BackoffIntervalConstants.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+
+using System;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration.Constants
+{
+    internal class BackoffIntervalConstants
```

Comment:
It's not obvious to see how this class is related to the failover. Should we just put all constants in one place?

Reply:
Renaming will help. It's clearer what scenario it's related to like the naming of other *Constants types. How about just `FailOverConstants`?